v1/api/swagger/components/schemas/ResourceBase.yaml

```diff
@@ -3,6 +3,8 @@ required:
   - ID
   - ETag
   - Name
+  - Created
+  - Modified
```

Comment:
I'm not clear if v1 API required Created & Modified timestamp.

Reply:
This is to avoid the `omitempty` tag